shoop/admin/locale/pt_BR/LC_MESSAGES/django.po

```diff
@@ -121,7 +157,7 @@ msgstr "Empresa"
 
 #, python-format
 msgid "User %(bind_user)s already has a contact"
-msgstr "Usuário %(bind_user)s já possui um contato"
+msgstr "O usuário %(bind_user)s já tem um contato"
```

Comment:
There is a control character (GS = group separator, i.e. 0x1D) in this line after `já` and before the space:
```
cat shoop/admin/locale/pt_BR/LC_MESSAGES/django.po | head -n160 | tail -n1 | hexdump -C
00000000  6d 73 67 73 74 72 20 22  4f 20 75 73 75 c3 a1 72  |msgstr "O usu..r|
00000010  69 6f 20 25 28 62 69 6e  64 5f 75 73 65 72 29 73  |io %(bind_user)s|
00000020  20 6a c3 a1 1d 20 74 65  6d 20 75 6d 20 63 6f 6e  | j... tem um con|
00000030  74 61 74 6f 22 0a                                 |tato".|
```